app/serializers/axiom_serializer.rb

```diff
@@ -0,0 +1,23 @@
+class AxiomSerializer < ApplicationSerializer
+  class Reference < ApplicationSerializer
+    attributes :iri
+    attributes :name
+
+    def iri
+      qualified_locid_for(object)
+    end
+  end
```

Comment:
could the need for this class Reference (which occurs in all the serializers) please be explained somewhere?